### DISCUSSION TITLE
Question about struct FlashSectors

### DISCUSSION THREAD

**Discussion by MarianSavchuk** [ARM-software/CMSIS_5#1370] [Q&A] [answered]
Hello,

I would like to know which value should be in the [AddrSector](https://github.com/ARM-software/CMSIS_5/blob/6368a853f29d641ad7a7759e17bee557ac26d8cd/Device/_Template_Flash/FlashOS.h#L39) field of the FlashSectors struct.
Is it an absolute address, or is it an offset relative to the [DevAdr](https://github.com/ARM-software/CMSIS_5/blob/6368a853f29d641ad7a7759e17bee557ac26d8cd/Device/_Template_Flash/FlashOS.h#L48) value from the FlashDevice struct?

Thanks in advance,
Marian

**Comment by RobertRostohar** [comment, marked as answer]
**AddrSector** should be specified as an offset relative to **DevAdr**.

**Reply by MarianSavchuk** [reply]
Thanks a lot for your answer, Robert!
Best regards
